homework/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <title>Javascript3</title>
+</head>
+
+<body>
+  <div>
+    <header class="header">
+      <div class="container">
+        <h1>HYF Repositories</h1>
+        <select id="repositories" class="repo-selector">
+          <option>Please choose a option </option>
+        </select>
+      </div>
+    </header>
+    <div id="main" class="container">
+
+      <div class="left-div whiteframe repo-properties">
+
+        <div><label>Repository:</label> <span id="repo-name"></span></div>
+
+        <div><label>Description:</label> <span id="repo-description"></span></div>
+
+        <div><label>Forks:</label> <span id="repo-forks"></span></div>
+
+        <div><label>Updated:</label> <span id="repo-updated"></span></div>
+
+      </div>
+      <div class="right-div whiteframe">
+      
+        <p class="contributor-header">Contributors</p> <span id="contributorElements"></span>
```

Comment:
You will receive a *list* (an array) of contributors. Displaying a list is typically done with `<ul>` (unordered list) element; with `<li>` elements inside of it. Then, for each contributor, you can have these `<p>` elements for their properties. But you'll probably want to construct those entirely in your code!